containers/python-examples/PyTorch/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.4"
+
+services:
+  server:
+    image: ghcr.io/deephaven/server-pytorch:${VERSION:-latest}
+    expose:
+      - '8080'
+    volumes:
+      - ./data:/data
+      - api-cache:/cache
+    environment:
+      - JAVA_TOOL_OPTIONS=-Xmx4g -Ddeephaven.console.type=python
+
+  web:
+    image: ghcr.io/deephaven/web:${VERSION:-latest}
+    expose:
+      - '80'
+    volumes:
+      - ./data:/data
+      - web-tmp:/tmp
+
+  grpc-proxy:
+    image: ghcr.io/deephaven/grpc-proxy:${VERSION:-latest}
+    environment:
+      - BACKEND_ADDR=grpc-api:8080
+    depends_on:
+      - grpc-api
+    expose:
+      - '8080'
+
+  envoy:
+    image: ghcr.io/deephaven/envoy:${VERSION:-latest}
+    depends_on:
+      - web
+      - grpc-proxy
+      - grpc-api
+    ports:
+      - "${PORT:-10000}:10000"
+
+  examples:
+    image: ghcr.io/deephaven/examples
```

Comment:
We should be explicit and use the `:latest` tag, instead of leaving it unspecified. Ditto in the other compose files.